### ISSUE TITLE
Transmission

### ISSUE THREAD

**Issue by Anzic23** [ioBroker/AdapterRequests#280]
It would be great to integrate transmission into iobroker. On npm there is a package for work on api https://www.npmjs.com/package/transmission . Maybe this will help.

**Comment by Apollon77** [comment]
Can you please add more infos what "transmission" is?

**Comment by misery** [comment]
Like the idea!

https://transmissionbt.com/

**Comment by DutchmanNL** [comment]
so if I understand correctly, you would like to have an adapter in iobroker to emulate an download client

**Comment by misery** [comment]
No, Transmission has a REST-API. Would be nice to support that to control transmission by remote.

https://github.com/transmission/transmission/blob/master/extras/rpc-spec.txt

https://www.npmjs.com/package/transmission is a nodejs-wrapper around that REST-API.

**Comment by DutchmanNL** [comment]
> No, Transmission has a REST-API. Would be nice to support that to control transmission by remote.

ah cheers now I get it thanks !
Yea the. API and NPM. module are straight forward. so integrating this is not that complex.
More a time issue to be honest, I could to it but no time currently for it.

I will assign my name not meaning I will handle it, but more : "If I have time left, a list with things I could pick up"

~Dutch
